operator-common/src/test/java/io/strimzi/operator/common/operator/MockCertManager.java

```diff
@@ -122,16 +122,104 @@ public class MockCertManager implements CertManager {
+            "kw/Pc+Q3Bze12/F66uMkas2Y7/gssyWgwVGhw9BmblronRMiLzuSc/ujFQGVQkWe\n" +
+            "yKRZ8flVqRwkdMFpmBjK/EWlz8C/7DFKsIccL0elNgjNQHGylRL6I0VwOxOEroUM\n" +
+            "Fr2d4vMWNvsybEmXTtXmomfJ8OOkfP/aIUMTdL/VS1RBfSG7GIpSRqtPKejykoAT\n" +
+            "GMeR4IAI4u17jqTu9X6vL4zll0fUNWzJVNPXrq7rTLvttdBxoOPObx1bcGWDUTEC\n" +
+            "gC0q/WEbE7AvDoDiy4eLcA++Ux40/w8cbS2Ng7PBPegu6nu7V2uYABjYGE/k6nSU\n" +
+            "MO/K/YcqehieqvE92OC7rOpUgb5fNe+Rsi0Mo/kRCPl6WeD7ORuCj06ti11n9oDZ\n" +
+            "8Xi7un6zI9MaWBc+oX65mEiW7mxJGv2+hmgS2evhQ5AFrNDSyRmTk9IHKxHJXIbR\n" +
+            "r+uA0QyFkLT5qAn7at9pAoIBAQDekDdGbVg7XIODOZUdimWiAylpk+RbrzHDFuKC\n" +
+            "px+VnXBdLAqbKCCw9/ry5QpOfIHRc2do52p9cymiL2+FtTD1F4PC4Mz3IIvikRXf\n" +
+            "JbPWo4U/3EKyZME5mbQx7cDXopOKsFYcG6Fq0ft6TDYakao+NgTuiOJ4z9aQFKi7\n" +
+            "KN2Dn6j4xzuZ2BxyDhYfAx+Kf/BsmSFb5GUNd1VXAwD3DofedsR1PqihXbg0qTpp\n" +
+            "PQza4uJPSQroEpyqcEtiWVAHCn8hcQ/MU8V868vmDdzGfXcebnyRDcfoc4B8gaJy\n" +
+            "YehikXjKIJ3i52ZH9QlWpoE6LZUW7XqENYrC4e8dv4/M/3MlAoIBAQDQZXzvyKJN\n" +
+            "km1vb7EUiq9o5KIZVXVLzCz+raTy4S5l6cKHlud4LjYaOMlAdmZwmRMNgi4EoKv/\n" +
+            "NoaZ7Qi04tHj9mft+RYkhkqhNfiRbuHK/nkLaG6AnnUJgadLiOKJrTz8aphEgl8H\n" +
+            "EfCbPdzhA089XuIYQ1dFc1/XucVUiwrwa9n3qmPOTy3/iNFmpUYERaxONK28ynPU\n" +
+            "A7PJ1lb9rI1Yd223NTjazH78EC4z0K4x+RQGf+vY4tJITdDLFAYOC4RJLeyRubYh\n" +
+            "cJ6/Y+syt2ybJU0ABGdD7z9WjQrFkH3y7zlDpJjI7cu9U9ohLnPRB2XRMb/QnrVN\n" +
+            "XmOVcVmkHaejAoIBAF1aPmUi/iw//MWxBHPUlhajnt4y80jlrki3M6E7ukrGM9hQ\n" +
+            "3c4Awd2Y1kDFg1IydT1GW6yrzIX8JQ1rOirqOjPDfjfilPZfR21sD7F+nfU8t2p2\n" +
+            "19791fJ0HxoGMKoB/kLg/e1iRY2JsCK3DsaMLM1VWTI4Zk3oV4YttCx00p0f/f8D\n" +
+            "imNYO243rF2rsLImQdE7Vbu2jBBe6oPJtcDq7mCIjpTjCFZO+jHUfbSgr+Cr4+L/\n" +
+            "rzxyehYfbe89qQybAYIka0z4IlDokqjXTbBP9gPksgkhr6U3tUAEHABAQvn9LJ4T\n" +
+            "mkYPa15pUFZxFqkxb2SjjY0g7CgcHnDq5cVFrKUCggEBAMBU+jK/Y/Pd8XxIhBEm\n" +
+            "Ue+oG2bwNTpIbTp7fmHeq5jJXo0PHAAR/kLDKBKDc9X1oNz1yYzh0PvREfccCc4Q\n" +
+            "DJCtCIxcODBrdJNGoWtv/5n+tC5wKpQwKmokrAcNha51IXUA2BlajWOUDhDozmMp\n" +
+            "T8efO2ITPIdPjSQnQtTdIrJzh46JRmXRM+3EEyb94BnvR7BB9Brqam5BIHxmgpDK\n" +
+            "4QTF7I+K4jclRwlR0aJmtW5PqEi7ZSsdY9/xWYMjrtblkCiN5ukzkcFZFFolWJvN\n" +
+            "J8IkceVTKw9KzpUNxORDj8pXOO2SCqReu4u3yPiYy7Eed7zTgmmDIGcQdy4kmAvc\n" +
+            "hqsCggEAWs3tK7cgOYJbbPGrW8rvtneK8+BH+15fyFjEZLv+0kDAu3pux5CBhsn8\n" +
+            "xHYe5sTUJnSMv4RFnGN777R56NKdqz+g6cOBwX8Wa9HJgUv37xQ0y+bpWwRGEcWa\n" +
+            "b40yeuAnBPwaGFwfm+IvB/h/Dovblwb8IDhQ3H0A/Xw94HWhEDhOfnDozd2zSaIS\n" +
+            "gPuuXmbT+5Nnefg6DupRxhOLUASG5eyaIJKGaBvoc2Ylmq5BXPyCnMIoUNS9e85q\n" +
+            "tUt1ndVBV2e7HV4BMXB6cCvE62H2ks8WoytpxOyg8GfwIeePe+seJOQAj6HuG4R8\n" +
+            "nEhbTwU8z4HvDeru2iLQfOv0JyKaXg==\n" +
+            "-----END PRIVATE KEY-----";
```

Comment:
Maybe you can give it a better validity? Eg. 100 years as with the other certs here?

Reply:
Done